src/core/utils/urlutils.js

```diff
@@ -119,9 +119,6 @@ export function filterParamsForExperienceLink (
   getComponentNamesForComponentTypes
 ) {
   const componentTypesToExclude = [
-    ComponentTypes.RANGE_FILTER,
-    ComponentTypes.DATE_RANGE_FILTER,
-    ComponentTypes.SORT_OPTIONS,
```

Comment:
Since we're removing everything except for GEOLOCATION_FILTER and FILTER_SEARCH, can we remove the if on L130?